rider/Digma.Rider.Plugin/Digma.Rider/Highlighting/CodeLensProviderFactory.cs

```diff
@@ -52,7 +55,10 @@ public BaseMethodInsightsProvider GetFactory(string lensTitle)
                 {
                     return _usageMethodInsightsProvider;    
                 }
-
+                if (lensTitle.ToUpper().Contains("SCALE"))
```

Comment:
is there any reason why we're using `contains` instead of `equals`?

Reply:
yes, because usually  lensTitle has longer text that contains some important word inside. So we must user contains as equals will not work here